src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateIsLiked}) => {
+  const entryClass = (id % 2 === 0) ? 'remote':'local';
+  const hearts = liked ? '❤️': '🤍' ;
+  
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className= { 'chat-entry' + ' ' + entryClass}>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time"> <TimeStamp time={timeStamp}/></p>
+        <button className="like" onClick={()=>updateIsLiked(id)}>{hearts}</button>
```

Comment:
Nice job using an anonymous function here to call the callback function on clicks. ✨